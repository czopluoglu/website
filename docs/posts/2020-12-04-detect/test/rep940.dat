11010011011000000011
10101101111111010011
10001001100110010110
10001000000000010010
00101001001000100010
01001001101011100010
10111101110010110110
00101111101010001010
00001100000100111111
10000000110110101010
11011111101111111110
11100111001000010011
11001111001111011111
00001110011110100111
00010101011011011110
11111110010111001111
00011101111100011011
10100011001011000111
11011101111111110111
01100001010010000011
11010100011010111110
00110011001110111110
10001011000000010111
11000001010010000111
10101011111110100111
00000000000000000110
00001001111011111111
11000011000010101010
00010111101111111111
10100101101011111111
11000100001000100010
11111111111000001011
10000011101010100011
11100101110000000110
11101111100110101110
10101010010010000110
00101000000000010011
11100001111111101111
00100101111110101111
00101001111111111111
00101111011110110010
01011110011110100111
01111110001001101100
10001111001111111110
01110000011100001010
00110101001011011111
11000100100100011000
00101000110000010011
11110110101001001010
00011011000101001111
11000110111111100010
11010001001111111110
11011000111001010111
01001101011110100010
11111001010100101110
11101111111111111111
01111000000000100011
00100110101110111110
10110111111011101111
00111001101001111111
11011111111111110111
01111110011000001000
01100110001101100011
10111000101100011111
01111001010101110110
10100001101000000111
01111101110010010010
11010101011000001001
11100000001110101000
10001111101111101010
00001101011010001101
01000001100010000111
11111111110001101011
11100100111111111110
10111100000100100011
00111111111111111111
10110101111010000000
11010100010000101010
00001101100110010111
01100011011111011111
10100101111111111111
01010111011101111110
11011000011110100101
10000001011001111101
11010000001111101011
00001111110010001011
10000000100000000111
11011001001110000000
01111101111101000111
00110101000011100110
01101101100111010111
00110000110100101101
01000101001010101011
01100010001101111111
00011000100110010111
10101101101101010111
10000101100110100111
10111111011111100110
00100001001000101010
11001001010110001011
01010001111010000011
00000000001110010101
01110111111101100011
10111011101001110111
10001001101011111111
01111001011100000111
01010001001010000110
11001010111110010111
10101111110100100010
00110101011010100110
11001101000101111011
11010001101110011111
10100101110110000110
01000010101001101011
01011110110100011011
10001111011111100011
01010001001111101111
00011101100000101100
11001101011110111111
10110110001111101011
10001001110010110101
11010101010011100010
10110101011111111111
10110100111101111100
00000001100010100010
11010000011110100001
10111001001101110110
11111011011010100001
00001001111010011110
11011110101000111111
10001000011100100111
11100000100110110111
00100101111010011010
10010101001000010101
11101011111111111110
00000001100100101001
00001000111101100110
10100000100001110011
11100101001110000111
10001100000010100000
11011011001001011110
01011101111000001111
01010001010010101011
00111001101111111111
00011111110001100000
10101101101110111111
11001000101110011111
11100010001111010111
01000001000011100100
10000001101010110111
11111101111100110111
01001011111111111101
00001000001001010011
10000001100111001011
10100010100010000110
01110001000011100011
01000010111010101111
01111101111110101010
01010011011100100110
00001001111111111110
11000001111001101110
01101011000010000000
10010000110111111110
00110101011011100100
01111111101110010100
00110111111000111111
11111000111111101111
01111110111101000000
00101000001001011110
01000101010001100011
11111111110011100110
10100001111011111010
10001001001111011110
11111000111101111111
11010110010010011000
00000101000110101001
11010101010111101111
10111110111000101100
11101101111101111101
01101001010011011110
10011101001001111110
00001001100111011010
10111001011010100110
10110000111101101100
10110110111001010100
00000010101010111111
11100101111110001111
00111111111111101010
00001001011010010110
01110001110011001110
11110110100010010111
10111000011010011011
10001000001011101110
10100001000010010010
01000110100010100010
00001101101111001011
00010000010000110010
11101101111111111010
10101000110100000100
00010001111100011110
00101101111010101010
10000100101011101001
00011010001011111110
10001000110110101110
00101011000000101010
11110000001110110111
11101101110111110110
01101001101110010100
11101010100110010011
11111011011110110011
10000101001000001111
01010100011111100110
00110001101110101111
10000010100110100010
10111011111011010000
10101101111111011001
10101101111000000011
11110111100110110101
00010101011001101111
00010010111110000000
01011101011010110010
00111111111011000011
10101101111001111010
11101100100010100011
11110011101010000100
10010001001100011001
10001101101110101001
01101001011110011011
00001001011010010110
10001010000110001110
00010101111100100011
00100101110000000010
10100111111110110111
01101100011110001010
01011100101111011111
10000001111110101110
11111011110010111111
10101101100001011000
11010100001110001011
00011001111110100101
11110111100110101100
10110011001101001010
00001001001111110100
11110111111111000100
00101001011000010101
10100101001110111110
01000101011011101110
11010101010110110111
11110111111010101011
10001100101111111111
