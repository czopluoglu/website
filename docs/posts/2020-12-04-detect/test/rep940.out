-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep940.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.00

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.97
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.8192
                   IDN index value:   0.6789
                           Ratio r:   0.6081

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    2    2    1    2    2    3    1    2
    4    1    3    4    4    1    4    1    3    1

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    5    9   12   16   18   20

   -----------CLUSTER  2-------------

    2    3    4    6    7   10

   -----------CLUSTER  3-------------

    8   13   19

   -----------CLUSTER  4-------------

   11   14   15   17
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d-+++++++-++--+---+-
   -d+++---+--+++------
   ++d-+++-+-+++------+
   ++-d++++------+-+-++
   ++++d++------+++++-+
   +-+++d+--+---++-+++-
   +-++++d+-------++---
   +--+--+d-++++-++----
   +++-----d+++++------
   -----+-++d+++---+++-
   +-+----+++d+++----++
   +++----++++d----++--
   -++----++++-d+--+-+-
   -+--++--+-+-+d++++--
   +--+++-+-----+d+--+-
   ----+-++-----++d-+++
   ---++++--+-+++--d+++
   ----++---+-+-+-++d++
   +--+-+---++-+-++++d+
   --+++-----+----++++d
 
 No cross validation for this DETECT run
